### ISSUE TITLE
Automatic scroll 

### ISSUE THREAD

**Issue by kiritoshi2002** [mihonapp/mihon#10]
### Describe your suggested feature

In the app called kotatsu very similar to tachiyomi there is an additional reading option called automatic scroll.is possible to add a similar features to mihon that would be very helpful 
![5](https://github.com/mihonapp/mihon/assets/68818865/8217ac01-f0e7-4aaa-831f-f140807b30dc)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an official extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/extensions/issues/new/choose).
- [X] I have updated the app to version **[0.15.3](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by dan-malprod** [comment]
This is a feature in the fork TachiyomiSY

**Comment by Sachit-10** [comment]
I can add this feature. Do you want me to add?

**Comment by Eien-Arashi** [comment]
> I can add this feature. Do you want me to add?

Please add this feature bro it will be useful

**Comment by vxyzview** [comment]
> > I can add this feature. Do you want me to add?
> 
> Please add this feature bro it will be useful

I think yea it will be useful if mihon have automatic scroll, I hope it will be added

> This is a feature in the fork TachiyomiSY

Not just TachiyomiSY, now Kotatsu have this feature too

**Comment by Rishi1232** [comment]
I just used kotatsu app and i loved this auto scroll feature I was gonna switch my manga reading apo but I don't want to leave mihon for its ui and categorization

**Comment by vxyzview** [comment]
> I just used kotatsu app and i loved this auto scroll feature I was gonna switch my manga reading apo but I don't want to leave mihon for its ui and categorization

Yes, kotatsu is good with the automatic scroll feature, but the UI/UX feels less smooth and light, for ui/ux and other feature mihon still the best.

**Comment by KaanAlper** [comment]
What is the status for now

**Comment by AntsyLich** [comment]
Not being worked on

**Comment by li64z** [comment]
How Is It going ?

**Comment by Smol-Ame** [comment]
@li64z if you had read the comments above before unnecessarily commenting (which pings the Mihon Discord channel & emails everyone subscribed to this issue), you'll see that it's "not being worked on".

Dont comment if you don't have anything useful information or contribution to the issue request, wait patiently, contribute the fix, or find somewhere else that'll work on demand for you.

**Comment by MajorTanya** [comment]
> How Is It going ?

1. Literally five days ago AntsyLich said this was not being worked on
2. Please don't post useless comments under inactive issues. Comments send out email notifications to all participants and to see an email come in only to see your unhelpful comment is annoying at best.
